src/icons/hicolor/scalable/apps/yast-checkmedia.svg

```diff
@@ -0,0 +1 @@
+<svg height="128" width="128" xmlns="http://www.w3.org/2000/svg"><linearGradient id="a" gradientUnits="userSpaceOnUse" x1="8" x2="120" y1="68" y2="68"><stop offset="0" stop-color="#c0bfbc"/><stop offset=".143" stop-color="#c0bfbc"/><stop offset=".25" stop-color="#deddda"/><stop offset=".5" stop-color="#c0bfbc" stop-opacity=".939"/><stop offset="1" stop-color="#c0bfbc" stop-opacity=".984"/></linearGradient><g stroke-width="4"><path d="M64 12C33.072 12 8 37.072 8 68s25.072 56 56 56 56-25.072 56-56-25.072-56-56-56zm0 36a20 20 0 0 1 20 20 20 20 0 0 1-20 20 20 20 0 0 1-20-20 20 20 0 0 1 20-20z" fill="url(#a)"/><path d="M64 8C33.072 8 8 33.072 8 64s25.072 56 56 56 56-25.072 56-56S94.928 8 64 8zm0 36a20 20 0 0 1 20 20 20 20 0 0 1-20 20 20 20 0 0 1-20-20 20 20 0 0 1 20-20z" fill="#deddda"/><path d="M64 44a20 20 0 0 0-20 20 20 20 0 0 0 20 20 20 20 0 0 0 20-20 20 20 0 0 0-20-20zm0 12a8 8 0 0 1 8 8 8 8 0 0 1-8 8 8 8 0 0 1-8-8 8 8 0 0 1 8-8z" fill="#c0bfbc" fill-opacity=".947"/><path d="M64 8v36a20 20 0 0 1 14.132 5.868L103.6 24.4A55.832 55.832 0 0 0 64 8zM49.868 78.132L24.4 103.6A55.832 55.832 0 0 0 64 120V84a20 20 0 0 1-14.132-5.868z" fill="#f6f5f4" opacity=".5"/><path d="M64 40a24 24 0 0 0-24 24 24 24 0 0 0 24 24 24 24 0 0 0 24-24 24 24 0 0 0-24-24zm0 4a20 20 0 0 1 20 20 20 20 0 0 1-20 20 20 20 0 0 1-20-20 20 20 0 0 1 20-20z" fill="#9a9996" fill-opacity=".947"/></g><path d="M64 40c-13.255 0-24 10.745-24 24s10.745 24 24 24a24.005 24.005 0 0 0 13.89-4.453l19.282 19.281a4 4 0 1 0 5.656-5.656L83.54 77.882A24 24 0 0 0 88 64c0-13.255-10.745-24-24-24zm0 8c8.837 0 16 7.163 16 16s-7.163 16-16 16-16-7.163-16-16 7.163-16 16-16z" fill="#3d3846"/></svg>
```

Comment:
A nit picking comment: This icon looks a bit strange to me, it rather looks like a volume control on a Hifi receiver or something like that. :grinning: 

Placing the magnifying glass exactly into the CD center does not look nice, IMHO moving it a bit out of the center and making it a bit asymmetric would help.

Reply:
LCP will likely redo the icon later to fit in with the new GNOME icon style. He said to remove the white squares in the corners on some of the icons.